systemtests/src/test/java/io/enmasse/systemtest/sharedinfra/SharedInfraDefaultsTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.systemtest.sharedinfra;
+
+import io.enmasse.api.model.MessagingInfra;
+import io.enmasse.api.model.MessagingInfraCondition;
+import io.enmasse.api.model.MessagingTenant;
+import io.enmasse.api.model.MessagingTenantCondition;
+import io.enmasse.systemtest.TestTag;
+import io.enmasse.systemtest.annotations.DefaultMessagingInfra;
+import io.enmasse.systemtest.annotations.DefaultMessagingTenant;
+import io.enmasse.systemtest.bases.TestBase;
+import io.enmasse.systemtest.bases.isolated.ITestIsolatedSharedInfra;
+import io.enmasse.systemtest.messaginginfra.resources.MessagingInfraResourceType;
+import io.enmasse.systemtest.messaginginfra.resources.MessagingTenantResourceType;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@Tag(TestTag.ISOLATED_SHARED_INFRA)
+public class SharedInfraDefaultsTest extends TestBase implements ITestIsolatedSharedInfra {
```

Comment:
We should really start to use a better name than SharedInfra. The fact that it is shared is kind of a coincidence, and in 1.0 it will be the only infra.

Reply:
Ok